### PR TITLE
Add kubeadm upgrade from stable to master test

### PR DESCRIPTION
**Pull request by jessicaochen**
Issue: https://github.com/kubernetes/kubeadm/issues/457

### CODE REVIEW

**Comment by cjwagner** [comment]
/approve
/approve cancel
(Just testing that self-approval has been re-enabled).

File: jobs/config.json [hunk 1]
```diff
@@ -7604,6 +7604,29 @@
       "sig-cluster-lifecycle"
     ]
   },
+  "ci-kubernetes-e2e-kubeadm-gce-upgrade-stable-master": {
```
Comment (luxas) [inline jobs/config.json]: call it `periodic-` as it is a periodic job

Comment (krzyzacy) [reply jobs/config.json]: @luxas all the other kubeadm jobs are with `ci-`, I think `ci-` is fine and we should simply avoid put `ci-` for postsubmit jobs

Comment (luxas) [reply jobs/config.json]: ok

**Pull request merged**